python/lsst/ts/salobj/base_csc.py

```diff
@@ -167,12 +167,6 @@ def __init__(
         if initial_state == State.FAULT:
             raise ValueError("initial_state cannot be FAULT")
 
-        # Postpone assigning command callbacks until `start` is done (but call
-        # assert_do_methods_present to fail early if there is a problem).
-        super().__init__(name=name, index=index, do_callbacks=True)
-
-        # Handle simulation_mode after constructing the base class,
-        # so that evt_simulationMode is available.
         if self.valid_simulation_modes is None:
             warnings.warn(
                 "valid_simulation_modes=None is deprecated", DeprecationWarning
```

Comment:
If this being set to None is deprecated, the attribute docs should be updated.

Reply:
Good suggestion. Done.